cdap-api/src/main/java/co/cask/cdap/api/dataset/table/Scanner.java

```diff
@@ -32,6 +32,7 @@
   /**
    * Closes the scanner and releases any resources.
    */
+  @Override
```

Comment:
Curious why we should have this method in this interface even though we extend AutoCloseable?